contents/tutorials/calculating-new-returning-users.md

```diff
@@ -0,0 +1,93 @@
+
+### First time event tracker
+
+You can set up and use the [first time event tracker app](/docs/apps/first-time-event-tracker) to add a property on users to say when they completed the pageview, identify, or custom event for the first time.
+
+To set it up, search for "First Time Event Tracker" in your PostHog instance, set the event you want to track in the configuration (like `$pageview`) and enable the app. 
+
+![First time event tracker](../images/tutorials/calculating-new-returning-users/first.png)
+
+You can then filter for events where `is_first_event_user` is `true` recently to get new users for analysis.
+
+### Manually set user property
+
+If neither of these works for you, you can set up a user property manually in a custom event. 
+
+For example, when a user first uses your API, set their creation date in their user properties. To do this, you can capture an event at the same time, and use the `$set_once` property like this:
+
+```python
+posthog.capture(
+    '<ph_project_api_key>',
+    event='new user created',
+    properties={ '$set_once': { 'created_at': '2023-04-24T22:02:02' } }
+)
+```
+
+This is especially useful for projects with many custom events, on the backend, not using autocapture, or not using `posthog-js`. 
+
+## Calculating returning users
+
+Unlike new users, there is one way to calculate returning users. It is identifying users who aren’t new but completed an event recently. To create a cohort for this, you can use the new user cohort as an anti-cohort (choose "Not in cohort"), and include another criteria for completing an event recently.
+
+![Returning users](../images/tutorials/calculating-new-returning-users/returning.png)
+
+If you created a `created_at` user property, you can also filter for users who weren’t created recently but completed the event. 
+
+### Using a retention insight
+
+If you care specifically about returning users, you can set up a retention insight. This shows the number of users who complete an event, then return to complete another event at a later date. You can customize the event and timeframe depending on what you care about.
+
+For example, an 8 week pageview retention graph looks like this:
+
+![Retention](../images/tutorials/calculating-new-returning-users/retention.png)
+
+You can learn more about retention in our [guide to churn analysis](/blog/customer-churn-analysis-guide).
+
+## Comparing new vs returning users
+
+With both new and returning users calculated, you can compare them for analysis. To do this, create an insight with two unique user trends: one filtered by the new user cohort, the other filtered by the returning user cohort. 
+
+![Versus](../images/tutorials/calculating-new-returning-users/versus.png)
+
+This gives you a trend similar to Google Analytics’ new vs returning visitors graph. You can also use these cohort filters for further analysis like comparing [session duration](/tutorials/session-metrics), feature usage, or even funnel conversion.
```

Comment:
Are many of our users familiar with Google Analytics graph? Is it worth mentioning?

Reply:
I think so